manifests/catalog-source.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
-  name: submariner-catalog
+  name: "$DOWNSTREAM_CATALOG_SOURCE"
```

Comment:
Please, return it back to be as `submariner-catalog`.
It gets overridden in any case during apply.
Putting here a variable could lead to false apply and override.